pages/draft-04/migration-notes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Migrating from Draft 03 to Draft 04
+section: docs
+---
+
+### Introduction
+
+The migration from Draft 3 to Draft 4 of JSON Schema introduces changes in how schemas are defined and validated. Draft 4, published on January 31, 2013, introduced new keywords and revised the behaviours of existing ones.
+
+This guide will help you adapt your JSON Schemas to align with Draft 4 requirements, covering keyword changes, updates, and behavioural modifications.
+
+### Keyword changelog
+
+Below is a summary table highlighting keyword changes between Draft 3 and Draft 4.
+
+| Keyword (Draft 3) | Keyword (Draft 4) | Specification | Keyword type      | Behavior Details                                                                                                                                                       |
+| ----------------- | ----------------- | ------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `$schema`         | `$schema`         | Core          | Identifier        | Change in the dialect (Uses the latest Draft4 dialect)                                                                                                                 |
```

Comment:
I'm not sure I agree with `$schema` being mentioned in these tables. The keyword behaviour remains exactly the same. It is just that the value changes to mention the new version?

Reply:
It wouldn't be great to just show up in the tutorial section if it's not on the table. I think we should leave this as is.